src/theme.ts

```diff
@@ -23,6 +23,7 @@ const theme = extendTheme(
         td: '#C7323A' /* TD red */,
       },
       primary: '#ffffff',
+      darkerBackground: '#11131d',
```

Comment:
This darker background color would be part of our `slate` palette, on the next line. Add it there instead under the `900` key.